api/environments/identities/views.py

```diff
@@ -21,6 +30,62 @@
 from util.views import SDKAPIView
 
 
+class EdgeIdentityViewSet(viewsets.ModelViewSet):
+    serializer_class = EdgeIdentitySerializer
+    permission_classes = [IsAuthenticated, NestedEnvironmentPermissions]
+    pagination_class = EdgeIdentityPagination
+    lookup_field = "identity_uuid"
+
+    @staticmethod
```

Comment:
I don't really like seeing static methods that are only marked as static because they don't use anything from the class. Static methods should be used outside of the class like `Class.static_method()`. Could / should we move the `dynamo_identifier_search_functions` constant into this class?

Reply:
hmm.. yeah, I was trying to hide internal filters of dynamodb from view layer, but I guess it's okay since we do that for django models as well.